ci-scripts/integration-test-stable.sh

```diff
@@ -115,4 +115,4 @@ elif [[ (-z $TEST || $TEST = "cli") && $CLI_FAIL -ne 0 ]]; then
 else
   exit 0
 fi
-# exit $FAIL
+# exit $FAIL
```

Comment:
Whitespaces should not pollute the PR diff. Make sure this file does not get listed .